draft-ietf-scitt-architecture.md

```diff
@@ -224,8 +223,7 @@ An Issuer may be the owner or author of software Artifacts, or an independent th
 
 Receipt:
 
-: a Receipt is a special form of COSE countersignature for Signed Statements that embeds cryptographic evidence that the Signed Statement is recorded in the Registry.
-A Receipt consists of a Registry-specific inclusion proof, a signature by the Transparency Service of the state of the Registry, and additional metadata (contained in the countersignature's protected headers) to assist in auditing.
+: a Receipt is a cryptographic proof that a Signed Statement is recorded in the Registry. Receipts are based on COSE Signed Merkle Tree Proofs {{-COSEMTP}}; they consist of a Registry-specific inclusion proof, a signature by the Transparency Service of the state of the Registry, and additional metadata (contained in the signature's protected headers) to assist in auditing.
```

Comment:
A Receipt is a SCITT word for the COMETRE concept of a "Signed Inclusion Proof".